<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-U9OHJGK" failures="0" tests="2" name="Default test" time="0.197" errors="0" timestamp="24 Dec 2019 21:28:39 GMT">
  <testcase classname="TestCases.MyQuoteTest" name="@BeforeClass loginBeforeTests" time="0.04">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestCases.MyQuoteTest.loginBeforeTests(MyQuoteTest.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass loginBeforeTests -->
  <testcase classname="TestCases.MyQuoteTest" name="@BeforeMethod setprop" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod setprop -->
  <testcase classname="TestCases.MyQuoteTest" name="@AfterMethod close" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod close -->
  <testcase classname="TestCases.MyQuoteTest" name="@BeforeMethod setprop" time="0.0">
    <skipped/>
  </testcase> <!-- @BeforeMethod setprop -->
  <testcase classname="TestCases.MyQuoteTest" name="@AfterMethod close" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterMethod close -->
  <testcase classname="TestCases.MyQuoteTest" name="selectRole" time="0.0">
    <skipped/>
  </testcase> <!-- selectRole -->
  <testcase classname="TestCases.MyQuoteTest" name="selectUser" time="0.0">
    <skipped/>
  </testcase> <!-- selectUser -->
</testsuite> <!-- Default test -->
